src/utils/plugins.py

```diff
@@ -0,0 +1,50 @@
+from utils import models
+
+
+class Plugin:
+    plugin_name = None
+    display_name = None
+    description = None
+    author = None
+    short_name = None
+    stage = None
+
+    manager_url = None
+
+    version = None
+    janeway_version = None
+
+    is_workflow_plugin = False
+    jump_url = None
+    handshake_url = None
+    article_pk_in_handshake_url = False
+    press_wide = False
+
+    kanban_card = '{plugin_name}/kanban_card.html'.format(
```

Comment:
where is this used?

Reply:
It will be used by the Typesetting Plugin.